scripts/variantstore/callset_QC/filter_set_samples_create_metrics.example.sql

```diff
@@ -66,7 +66,7 @@ SELECT "$NAME_OF_FILTER_SET" filter_set_name,
                CASE WHEN INSTR(call_GT, "0") > 0 THEN "het" ELSE "homvar" END as gt_type, -- if GT contains a zero, its a het
                titv(ref, alt) as titv,
                CASE WHEN gnomad.location IS NULL THEN false ELSE true END in_gnomad
-        FROM `$FQ_DATASET.vet_all` v
+        FROM `$FQ_VET_ALL` v
```

Comment:
I think you either need to add FQ_VET_ALL to the README instructions or use a variable like:  $FQ_PREFIX_vet_all